src/lava/magma/compiler/channels/selector.py

```diff
@@ -0,0 +1,17 @@
+# Copyright (C) 2021-22 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+
+import typing as ty
+from lava.magma.runtime.message_infrastructure import RecvPort
+
+
+class Selector:
```

Comment:
Could we please check if the Selector behavior has changed in case of no communication was received on any channels?

Reply:
The Selector behavior has some changes with CspSelector. Original CspSelector need to set observer to RecvPort. But our RecvPort has implemented in msglib and remove the observer object in cpp side. So we want to do some changes with CspSelector